AUTO_RECONSCTRUCT_BY_JSON_SCHEME.md

```diff
@@ -58,46 +58,49 @@ add_a_car(car: Car) {
   * for `array` type, we need to declare it in the format of `{array: {value: valueType}}`
   * for `map` type, we need to declare it in the format of `{map: {key: 'KeyType', value: 'valueType'}}`
 * Custom Class types: `Car` or any class types
-* Near collection types: `Vector`, `LookupMap`, `LookupSet`, `UnorderedMap`, `UnorderedSet`  
+* Near collection class types: `Vector`, `LookupMap`, `LookupSet`, `UnorderedMap`, `UnorderedSet`, need to declare in the format of `{class: ClassType, value: ValueType}`
+  * we can ignore `value` field if we don't need to auto reconstruct value to specific class. we often ignore `value` field if value tye are `string`, `number`, `boolean`
 We have a test example which contains all those types in one schema: [status-deserialize-class.js](./examples/src/status-deserialize-class.js)
 ```js
-class StatusDeserializeClass {
-    static schema = {
-        is_inited: "boolean",
-        records: {map: {key: 'string', value: 'string'}},
-        car: Car,
-        messages: {array: {value: 'string'}},
-        efficient_recordes: {unordered_map: {value: 'string'}},
-        nested_efficient_recordes: {unordered_map: {value: {unordered_map: {value: 'string'}}}},
-        nested_lookup_recordes: {unordered_map: {value: {lookup_map: {value: 'string'}}}},
-        vector_nested_group: {vector: {value: {lookup_map: {value: 'string'}}}},
-        lookup_nest_vec: {lookup_map: {value: {vector: {value: 'string'}}}},
-        unordered_set: {unordered_set: {value: 'string'}},
-        user_car_map: {unordered_map: {value: Car}},
-        big_num: 'bigint',
-        date: 'date'
-    };
-
-    constructor() {
-        this.is_inited = false;
-        this.records = {};
-        this.car = new Car();
-        this.messages = [];
-        // account_id -> message
-        this.efficient_recordes = new UnorderedMap("a");
-        // id -> account_id -> message
-        this.nested_efficient_recordes = new UnorderedMap("b");
-        // id -> account_id -> message
-        this.nested_lookup_recordes = new UnorderedMap("c");
-        // index -> account_id -> message
-        this.vector_nested_group = new Vector("d");
-        // account_id -> index -> message
-        this.lookup_nest_vec = new LookupMap("e");
-        this.unordered_set = new UnorderedSet("f");
-        this.user_car_map = new UnorderedMap("g");
-        this.big_num = 1n;
-        this.date = new Date();
-    }
+export class StatusDeserializeClass {
+  static schema = {
+    is_inited: "boolean",
+    records: {map: { key: 'string', value: 'string' }},
+    truck: Truck,
+    messages: {array: {value: 'string'}},
+    efficient_recordes: {class: UnorderedMap},
```

Comment:
Can this be simply `efficient_recordes: UnorderedMap`, or skipped in schema

Reply:
simplied to `efficient_recordes: UnorderedMap`